app/assets/stylesheets/application.scss

```diff
@@ -8,3 +8,4 @@
 @import "flash_messages";
 @import "sign_in";
 @import "sign_up";
+@import "shared/wishlist";
```

Comment:
This SCSS partial is not shared along the all app's view. No need to put it in `shared`.